
<launch>    
    <node pkg="multiespectral_fb" type="multiespectral_control.py" name="multiespectral_control" output="screen">
        <param name="log_level" value="debug"/>
    </node>
</launch>